src/renderer/components/+custom-resources/crd-resource-details.tsx

```diff
@@ -12,67 +12,83 @@ import { KubeObjectDetailsProps } from "../kube-object";
 import { crdStore } from "./crd.store";
 import { KubeObjectMeta } from "../kube-object/kube-object-meta";
 import { Input } from "../input";
-import { CustomResourceDefinition } from "../../api/endpoints/crd.api";
+import { AdditionalPrinterColumnsV1, CustomResourceDefinition } from "../../api/endpoints/crd.api";
 
 interface Props extends KubeObjectDetailsProps<CustomResourceDefinition> {
 }
 
-function CrdColumnValue({ value }: { value: any[] | {} | string }) {
+function convertSpecValue(value: any): any {
   if (Array.isArray(value)) {
-    return <>{value.map((item, index) => <CrdColumnValue key={index} value={item} />)}</>
+    return value.map(convertSpecValue)
   }
-  if (typeof(value) === 'object') return (
-    <Input
-      readOnly
-      multiLine
-      theme="round-black"
-      className="box grow"
-      value={JSON.stringify(value, null, 2)}
-    />
-  );
-  return <span>{value}</span>;
+
+  if (typeof value === "object") {
+    return (
+      <Input
+        readOnly
+        multiLine
+        theme="round-black"
+        className="box grow"
+        value={JSON.stringify(value, null, 2)}
+      />
+    )
+  }
+
+  return value
 }
+
 @observer
 export class CrdResourceDetails extends React.Component<Props> {
   @computed get crd() {
     return crdStore.getByObject(this.props.object);
   }
 
+  renderAdditionalColumns(crd: CustomResourceDefinition, columns: AdditionalPrinterColumnsV1[]) {
+    return columns.map(({ name, jsonPath: jp }) => (
+      <DrawerItem key={name} name={name} renderBoolean>
```

Comment:
Maybe set `renderBoolean` `true` by default so no need to explicitly set this prop? Wdyt?

Reply:
We could definitely do that. However I am a bit on the fence about it. If we make rendering boolean's default it makes the coding easier but we might then get random booleans being rendered. If keep it false then we have to add `renderBoolean` to a bunch of places but we are much less likely to get unwanted booleans being rendered.